.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,32 @@
+name: Node CI
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
```

Comment:
For debugging purpose, push to master triggers this workflow. (removed in later commits)